lib/api/controllers/documentController.js

```diff
@@ -133,6 +134,38 @@ class DocumentController extends NativeController {
     return this.ask('core:storage:public:document:exist', index, collection, id);
   }
 
+  /**
+   * @param {Request} request
+   * @returns {Promise<Boolean>}
+   */
+  async mExists (request) {
+    let ids;
+    if ( request.input.body
+      && request.input.body.ids
+      && Object.keys(request.input.body.ids).length
+    ) {
+      ids = request.getBodyArray('ids');
+    }
+    else {
+      ids = request.getArray('ids');
+    }
+    const { index, collection } = request.getIndexAndCollection();
+    
+    this.assertNotExceedMaxFetch(ids.length);
+    const strict = request.getBoolean('strict');
```

Comment:
Nitpicking: group line of code doing the same semantic steps, here first we are extracting parameters from the request, then we can checks limits
```suggestion
    const { index, collection } = request.getIndexAndCollection();
    const strict = request.getBoolean('strict');
    
    this.assertNotExceedMaxFetch(ids.length);
```

Reply:
@matissepage